exercises/bank-account/canonical-data.json

```diff
@@ -0,0 +1,372 @@
+{
+  "exercise": "bank-account",
+  "comments": [
+    "This exercise is a good candidate to practice concurrency, which",
+    "may not be available or possible in the implementing language track.",
+    "In that case, all tests can be executed on a single thread. If there",
+    "are tests that don't work using a single thread, these should be",
+    "marked with a concurrency scenario.",
```

Comment:
I think this comment should, in a slightly different form, be moved to the "concurrent" test case.